src/styles/blocks/product-card.scss

```
@@ -0,0 +1,92 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-grey;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $text-grey;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    height: 40px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 18px;
+    text-transform: uppercase;
+
+    color: #fff;
+
+    background-color: $blue-color;
+    transition: all 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```